frontend/src/api/postProjects.js

```diff
@@ -0,0 +1,40 @@
+import ApplicationContext from "../applicationContext/ApplicationContext";
+
+export const postProject = async (item) => {
+    const api_version = ApplicationContext.get().helpers().backEndConfig.apiVersion;
+
+    const newProject = { ...item };
+
+    // TODO: Something something project type.
```

Comment:
Is there something that needs to be done here still?

Reply:
Yes, but after our demo. Basically we're not sending/storing the project type or using it at all on the backend. We only know of Research projects. So... Tim and I went with just skipping actually building that for now.